spec/models/authenticator/amazon_spec.rb

```diff
@@ -124,12 +124,12 @@ class FakeGroup < Struct.new(:name)
   end
 
   before(:each) do
-    allow_any_instance_of(described_class).to receive(:aws_connect) { |*args| FakeAmazon.new(user_data, *args) }
+    allow_any_instance_of(described_class).to receive(:aws_connect) { |instance, *args| FakeAmazon.new(user_data, *args) }
```

Comment:
In RSpec 3 the instance is always the first argument to the block. Gah, yes it should be `_instance` since it's unused, I forgot to do that when I changed all the refs in https://github.com/chrisarcand/manageiq/commit/15f8a9f7207a800d48af341bbb903f8d4b53a558 Good catch